src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -636,6 +717,34 @@ public static void AppliedRequestFormLimits(this ILogger logger)
             }
         }
 
+        private static void ExecutingFilters(ILogger logger, string filterType, IEnumerable<IFilterMetadata> filters)
+        {
+            string[] filterList = null;
+            if (filters.Any())
+            {
+                filterList = GetFilterList(filters);
+            }
+
+            _executingFilters(logger, filterType, filterList, null);
+        }
+
+        private static string[] GetFilterList(IEnumerable<IFilterMetadata> filters)
+        {
+            var filterList = new List<string>();
+            foreach (var filter in filters)
+            {
+                if (filter is IOrderedFilter orderedFilter)
+                {
+                    filterList.Add($"{filter.GetType()}(Order:{orderedFilter.Order})");
```

Comment:
I recommend padding some spaces in here, e.g. `{filter.GetType()} (Order: {orderedFilter.Order})`